tests/tests_mot/tracker/test_camera_flow.py

```diff
@@ -0,0 +1,36 @@
+from mot.tracker import video_utils
+from mot.tracker import camera_flow
+import os
+import numpy as np
+import shutil
+import cv2
+
+home = os.path.expanduser("~")
+PATH_TO_TEST_VIDEO = os.path.join(home, ".mot/tests/test_video.mp4")
+
+PATH_TO_OUTPUT_SPLIT = "/tmp/test_outut_video/"
+
+def test_camera_flow():
+    if os.path.isdir(PATH_TO_OUTPUT_SPLIT):
```

Comment:
pytest provide convenient fixtures in order to have a tmpdir. You can used it as follow.

```python
 def test_camera_flow(tmp_dir):
```